h2o-py/h2o/h2o.py

```diff
@@ -871,10 +871,10 @@ def parse_setup(raw_frames, destination_frame=None, header=0, separator=None, co
     if column_names is not None:
         if not isinstance(column_names, list): raise ValueError("col_names should be a list")
         if (skipped_columns is not None) and len(skipped_columns)>0:
-            if (len(column_names)) != parse_column_len:
+            if ((len(column_names)-len(skipped_columns)) != parse_column_len) and (len(column_names) != parse_column_len):
```

Comment:
can you explain why with the current behaviour, `parse_column_len` must equal one or the other? why do we not always enforce `len(column_names)-len(skipped_columns) == parse_column_len` for example?
Especially given that this test is applied only if `len(skipped_columns)>0` and given that above we define:
```python
    parse_column_len = len(j["column_types"]) if skipped_columns is None else (len(j["column_types"])-len(skipped_columns))
```
therefore I don't understand the case where we would have `len(column_names) == parse_column_len`, as checked in the original test.

Should we not only test :
```
if (len(column_names)-len(skipped_columns)) != parse_column_len:
```
and if not, doesn't it show a more profound inconsistency/bug?

Reply:
If skipped_columns is not null, then len(column_names)-len(skipped_columns) should equal to parse_column_len.

I am not sure why I put the second condition there.  I removed it.